Source/UI/BillingForm/ViewController/BillingFormViewController.swift

```diff
@@ -135,47 +156,47 @@ final class BillingFormViewController: UIViewController {
                                       keyboardWillShow: #selector(keyboardWillShow),
                                       keyboardWillHide: #selector(keyboardWillHide))
     }
+
+    @objc func cancelButtonIsPressed() {
+        delegate?.cancelButtonIsPressed(sender: self)
+    }
+
+    @objc func doneButtonIsPressed() {
+        delegate?.doneButtonIsPressed(sender: self)
+    }
 }
 
 // MARK: - Views Layout Constraint
 
 extension BillingFormViewController {
+    private func setupNavigationBar() {
+        navigationController?.navigationBar.tintColor = viewModel.style.header.cancelButton.activeTintColor
```

Comment:
Should the navigation bar tint get the color from one of the buttons? What if they have different colours for each button??